NN_2L2H2O_poly2d/NN_2L2H2O_poly2d_benchmarking_result.md

```diff
@@ -0,0 +1,23 @@
+# Benchmarking result of NN_2L2OH2O_poly2d 
+
+### This file contains :  
+
+#### 1. the benchmarking result of `NN_2L2OH2O_poly2d` in Neural Network model on 42105 samples. This benchmarking is made to the part of the code *predicting the final score* of all samples, excluding *loading parameters, initializing layers and GPU resource context, etc*. The result records the run time on different machine/GPU, supporting library version, driver version, compiler version etc. 
+#### 2. the benchmarking result of calculating potential energy `E2poly` in polynomial method. It is made on the part of the code *calculating the potential energy E2poly* in the file *mbpol.cpp* only. Note the run time is assessed for **each sample**, as the code can only take in one sample to make the calculation. So, the time comparable with what is recored in the #1 is *run_time_per_sample[=Time_per_run] x sample_count[=42105 samples]*
+
+<br>
+
+Code Language  |  Machine  |  GPU ID  |  Iterations |  Total time [s]  |  **Time per run [s]**  |  GPU Type  |  Driver Version  |  Compiler  |  Compiler version  |  Supporting Lib 1  |  Supporting Lib 1 version  |  Supporting Lib 2  |  Supporting Lib 2 version  |  Supporting Lib 3  |  Supporting Lib 3 version  |  Supporting Lib 4  |  Supporting Lib 4 version
```

Comment:
too many columns. make this 1 single column:

Compiler:
g++	4.8.5	CUDA	8.0.61	CUDNN	6.0.21

also try to keep number of row to a minimum.
for example on Huey it looks like the 2 GPUs are the same, report only 1